android/app/src/main/java/com/mattermost/rnbeta/MainApplication.java

```diff
@@ -55,20 +55,22 @@ public boolean isDebug() {
             new SvgPackage(),
             new LinearGradientPackage(),
             new OrientationPackage(),
-            new RNNotificationsPackage(MainApplication.this)
+            new RNNotificationsPackage(this),
+            new LocalAuthPackage(),
+            new JailMonkeyPackage(),
+            new MattermostManagedPackage()
     );
   }
 
   @Override
   public void onCreate() {
```

Comment:
I assume that this is called right after the constructor? Just want to make sure that `notificationsLifecycleFacade` is always going to be initialized before somebody tries to access it, because it is public.